angular/projects/admin/src/app/admin/document-type/document-type-form/document-type-form.component.ts

```diff
@@ -151,9 +153,7 @@ export class DocumentTypeFormComponent implements OnInit, OnDestroy, OnChanges {
       fields: this.fieldForms.value,
       standalone: formData.standalone
     } as DocumentType);
-
-    if (this.afterSaveRoute) {
-      this.router.navigateByUrl(this.afterSaveRoute);
-    }
+    this.snackBar.open('Saved', 'Dismiss', {duration: 2000});
+    this.router.navigateByUrl(this.onCancelRoute);
```

Comment:
This always routes to `onCancelRoute`.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/93/commits/6a9acaab98950253b291aee3857c65a86d359108